sysvinit/openbsd/cloudinitlocal.tmpl

```diff
@@ -0,0 +1,40 @@
+## template:jinja
+#!/bin/ksh
+
+# PROVIDE: cloudinitlocal
+# REQUIRE: NETWORKING
+
+# After NETWORKING because we don't want staticroute to wipe
+# the route set by the DHCP client toward the meta-data server.
+
+daemon="cloud-init"
+daemon_execdir="{{prefix}}/bin"
+daemon_flags="init -l"
+daemon_rtable=0
+daemon_timeout=1
+daemon_user=root
+
+. /etc/rc.d/rc.subr
+
+rc_bg="YES" # (undefined or "YES")
+rc_usercheck="YES"
+
+rc_start() {
+    if test -e /etc/cloud/cloud-init.disabled; then
```

Comment:
this file should probably be in `{{prefix}}/etc… `

Reply:
I did not do this because `etc` is at `/etc` not `/usr/local/etc`